plugins/recommendations-plugin/src/plugin/recommendations-plugin.ts

```diff
@@ -127,6 +127,7 @@ export class RecommendationsPlugin {
   async afterClone(): Promise<void> {
     // current workspaces
     const workspaceFolders = theia.workspace.workspaceFolders || [];
+    this.outputChannel.appendLine(`workspaceFolders=${JSON.stringify(workspaceFolders)}`);
```

Comment:
It would help to say what this line is describing: "Workspace folders after clone: workspaceFolder=..."

Reply:
fixed